lib/cleanupDone.js

```diff
@@ -1,6 +1,11 @@
 function cleanupDone(config, sftp) {
   var root = config.fileDownloadDir + '/' + config.completedDir
   return sftp.list(root)
+    .catch(err => {
```

Comment:
Hmm, so we skip _all_ errors that occur whilst listing the contents of the directory? That doesn't quite feel right.

Reply:
I thought of that. Although I'm not really sure what else can be done, at least it logs the error.

It seems nice that it continues processing whatever it can.

Reply:
Yeah true